openpype/tools/pyblish_pype/app.py

```diff
@@ -98,7 +98,11 @@ def show(parent=None):
 
         self._window.show()
         self._window.activateWindow()
-        self._window.setWindowTitle(settings.WindowTitle)
+        env_title = os.getenv("PYBLISH_TITLE")
+        if env_title:
+            self._window.setWindowTitle(env_title)
+        else:
+            self._window.setWindowTitle(settings.WindowTitle)
```

Comment:
Why need the environment variable if you could also tweak `settings.WindowTitle`? Or are you running pyblish_pype in a subprocess?